### DISCUSSION TITLE
How to make "tab" and "shift-tab" switch the selection in  command palette ?

### DISCUSSION THREAD

**Discussion by Dangerise** [zed-industries/zed#14848] [Feature Requests]
I want to bind my "tab" to act like "down" and bind my " shift-tab" to act like "up" , so that I can move the selection by "tab" and "shift-tab" in the command palette or a file finder 

How to do that ?

**Comment by PlexSheep** [comment]
I found what to do!

```json
    {
        "bindings": {
            "shift-tab": "menu::SelectPrev",
            "tab": "menu::SelectNext",
        }
    },
```

Just this (no context) works for me

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
